copilot/kubernetes-argocd-passthru.py

```
@@ -0,0 +1,103 @@
+            body=body,
+            _preload_content=False
+        )
+        
+        return {
+            "status_code": response.status,
+            "headers": dict(response.headers),
+            "body": yaml.safe_load(response.data)
+        }
+    
+    except ApiException as e:
+        raise HTTPException(status_code=e.status, detail=str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)